webrtc.html

```diff
@@ -2356,7 +2357,13 @@
 
           <dd>This indicates the index (starting at zero) of the m-line in the
           SDP this candidate is associated with.</dd>
+        </dl>
+        <div class="note">
+          <p>The constructor on <code><a>RTCIceCandidate</a></code>
+          exists for legacy compatibility reasons only.</p>
+        </div>
 
+        <dl class="idl" title="dictionary RTCIceCandidateFull : RTCIceCandidate">
```

Comment:
I must admit to a dislike for the name "RTCIceCandidateFull" (only partly because Norwegian associates the word "Full" with "drunk").
Can we make it RTCIceCandidateInformation, to emphasize that this is used to give information about a candidate rather than initializing from it?
(The way dictionaries work, sending an RTCIceCandidateInformation into an interface that expects an RTCIceCandidate will Just Work, btw).
</bikeshed>